### ISSUE TITLE
uyuni-docs_en not able to upgrade to new version - dependency mismatch 

### ISSUE THREAD

**Issue by lumarel** [uyuni-project/uyuni#5419]
# Problem description

While upgrading from 2022.03 to 2022.05 a dependency mismatch appeared:

```
# zypper -vv up
Verbosity: 3
Initializing Target
Loading repository data...
Reading installed packages...
Force resolution: No

The following 5 package updates will NOT be installed:
  Uyuni-Server-release          2022.05-180.1.uyuni1  x86_64  Uyuni Server Stable                                           obs://build.opensuse.org/systemsmanagement:Uyuni
  netty                         4.1.75-150200.4.9.1   x86_64  Update repository with updates from SUSE Linux Enterprise 15  SUSE LLC <https://www.suse.com/>
  prometheus-postgres_exporter  0.10.0-150000.1.3.1   x86_64  Update repository with updates from SUSE Linux Enterprise 15  SUSE LLC <https://www.suse.com/>
  uyuni-docs_en                 2022.05-1.1.uyuni1    noarch  Uyuni Server Stable                                           obs://build.opensuse.org/systemsmanagement:Uyuni
  uyuni-docs_en-pdf             2022.05-1.1.uyuni1    noarch  Uyuni Server Stable                                           obs://build.opensuse.org/systemsmanagement:Uyuni
```

As far as I could see the `uyuni-docs_en*` packages were not able to install because of the `uyuni-doc-indexes` package:

```
# zypper in uyuni-docs_en uyuni-docs_en-pdf
Loading repository data...
Reading installed packages...
Resolving package dependencies...

Problem: the installed uyuni-doc-indexes-2022.03-2.11.uyuni1.noarch requires 'uyuni-docs_en = 2022.03', but this requirement cannot be provided
  deleted providers: uyuni-docs_en-2022.03-1.1.uyuni1.noarch
 Solution 1: Following actions will be done:
  do not install uyuni-docs_en-2022.05-1.1.uyuni1.noarch
  do not install uyuni-docs_en-pdf-2022.05-1.1.uyuni1.noarch
 Solution 2: deinstallation of uyuni-doc-indexes-2022.03-2.11.uyuni1.noarch
 Solution 3: break uyuni-doc-indexes-2022.03-2.11.uyuni1.noarch by ignoring some of its dependencies
```

## Version of Uyuni Server and Proxy (if used)

```
Information for package Uyuni-Server-release:
---------------------------------------------
Repository     : Uyuni Server Stable
Name           : Uyuni-Server-release
Version        : 2022.05-180.1.uyuni1
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes (automatically)
Status         : out-of-date (version 2022.03-172.4.uyuni1 installed)
Source package : Uyuni-Server-release-2022.05-180.1.uyuni1.src
Summary        : Uyuni Server
Description    :
    Uyuni lets you efficiently manage physical, virtual,
    and cloud-based Linux systems. It provides automated and cost-effective
    configuration and software management, asset management, and system
    provisioning.
```

@juliogonzalez as we talked in Gitter 👌🏻

**Comment by juliogonzalez** [comment]
Fixed by https://build.opensuse.org/package/rdiff/systemsmanagement:Uyuni:Master/uyuni-docs_en?linkrev=base&rev=49

It will be part of 2022.06.

For now, ` Solution 2: deinstallation of uyuni-doc-indexes-2022.03-2.11.uyuni1.noarch` can be used as workaround.

**Comment by sbluhm** [comment]
can confirm `Solution 2` works. Also got the same result triggering `zypper up uyuni-docs_en`.

**Comment by juliogonzalez** [comment]
> can confirm `Solution 2` works. 

Yes, for now that's the solution at Uyuni 2022.05 (Stable)

> Also got the same result triggering `zypper up uyuni-docs_en`.

But I guess you are on master/development, right? The fix was not published to Stable, as we can't publish individual packages.

**Comment by sbluhm** [comment]
This was 2022.05 stable. Was just trying to say that `zypper install`as well as `zypper update` bring up the "Solution" choice.

**Comment by juliogonzalez** [comment]
> This was 2022.05 stable. Was just trying to say that `zypper install`as well as `zypper update` bring up the "Solution" choice.

Ah, yeah, right. That's what's needed :-)

**Comment by parlt91** [comment]
@lumarel A fix was released as mentioned in https://github.com/uyuni-project/uyuni/issues/5419#issuecomment-1123938257
Please close the issue if you see the case as resolved.

**Comment by juliogonzalez** [comment]
> @lumarel A fix was released as mentioned in [#5419 (comment)](https://github.com/uyuni-project/uyuni/issues/5419#issuecomment-1123938257) Please close the issue if you see the case as resolved.

No, not released yet. As it's said there, it will be part of 2022.06. For now, the workaround at that comment must be applied.

**Comment by lumarel** [comment]
@juliogonzalez Sorry didn't have the time earlier, can confirm that the workaround works:

```
zypper remove uyuni-doc-indexes-2022.03-2.11.uyuni1.noarch
zypper up
```
